src/robot/libdocpkg/model.py

```diff
@@ -13,13 +13,39 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-from itertools import chain
 import json
 import re
+from itertools import chain
+
+try:
+    from enum import Enum
+
+    EnumType = type(Enum)
+except ImportError:  # Standard in Py 3.4+ but can be separately installed
+    class EnumType(object):
+        pass
+
+try:
+    from typing import TypedDict
+
+    TypedDictType = type(TypedDict('TypedDictDummy', {}))
+except ImportError:
+    class TypedDictType(object):
+        pass
+
+try:
+    from typing_extensions import TypedDict as ExtTypedDict
+
+    ExtTypedDictType = type(ExtTypedDict('TypedDictDummy', {}))
+except ImportError:
+    class ExtTypedDictType(object):
+        pass
```

Comment:
Are these imports needed anymore? If they are, couldn't they be imported from the new `datatypesdoc` module?